packages/amplify_datastore/ios/Classes/types/hub/FlutterOutboxMutationEnqueuedEvent.swift

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Foundation
+import Amplify
+import AmplifyPlugins
+import AWSCore
+import Combine
+
+struct FlutterOutboxMutationEnqueuedEvent: FlutterHubEvent {
+    var eventName: String
+    var modelName: String
+    var element:  FlutterHubElement?
+    
+    init(outboxMutationEnqueued: OutboxMutationEvent, eventName: String, schema: ModelSchema) throws {
+
+
+        self.eventName = eventName
+        self.modelName = outboxMutationEnqueued.modelName
+        do {
+            self.element = try FlutterHubElement(hubElement: outboxMutationEnqueued.element, schema: schema)
+        } catch {
+            throw error
+        }
```

Comment:
If the element is optional, why throw an error if we can't find a value for it.

Reply:
Fixed.